src/blob/controller.ts

```diff
@@ -13,6 +13,10 @@ class BlobController extends Controller {
         ctx.response.body = fs.readFileSync(__dirname + '/../../assets/' + ctx.state.params.filename);
         ctx.response.type = 'text/css';
         break;
+      case 'webauthn.js' :
```

Comment:
Really need to finish the `@curveball/static` package so we don't need this.

Reply:
++